www/templates/character.html

```diff
@@ -0,0 +1,38 @@
+<ion-view view-title="Character">
+  <ion-content>
+      <div class="list">
+          <label class="item item-input">
+              <input type="text" placeholder="Name">
+          </label>
+          <label class="item item-input">
+              <input type="text" placeholder="Race">
+          </label>
+          <label class="item item-input">
+              <textarea placeholder="Class"></textarea>
+          </label>
+          <label class ="item item-input">
+              <input type="text" placeholder="skill rank" />
+              <input type="text" placeholder="level" />
```

Comment:
Nice work! However I noticed you used input type="text" for everything, did you know you can do a lot with that tag? here is the link for all the avaliable types: http://www.w3schools.com/tags/att_input_type.asp